powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/entity/ActivationHistoryEntity.java

```diff
@@ -17,14 +17,24 @@
  */
 package io.getlime.security.powerauth.app.server.database.model.entity;
 
-import io.getlime.security.powerauth.app.server.database.model.ActivationStatus;
-import io.getlime.security.powerauth.app.server.database.model.ActivationStatusConverter;
-
-import javax.persistence.*;
 import java.io.Serializable;
 import java.util.Date;
 import java.util.Objects;
 
+import javax.persistence.Column;
```

Comment:
Máte někde sdílené nastavení IDE nejen ohledně organizace importů?

Reply:
We use default IntelliJ IDEA settings for imports.

Reply:
Ok, I reverted to IDE default code style.